operatorapi/consts.go

```diff
@@ -21,6 +21,7 @@ const (
 	ConsoleSubnetLicense   = "CONSOLE_SUBNET_LICENSE"
 	ConsoleOperatorSAToken = "CONSOLE_OPERATOR_SA_TOKEN"
 	MinIOSubnetLicense     = "MINIO_SUBNET_LICENSE"
+	ConsoleMPMode          = "CONSOLE_OPERATOR_MP_MODE"
```

Comment:
```suggestion
	ConsoleMPMode          = "CONSOLE_OPERATOR_MARKETPLACE"
```

Is enough there is no need for MP_MODE